build.gradle

```diff
@@ -41,31 +52,40 @@ sourceSets {
             srcDirs('build/nativeLib/')
         }
     }
+    examples {
+        java
+    }
 }
 
 jar {
     from sourceSets.main.output
 }
 
 repositories {
+    mavenLocal()
     mavenCentral()
 }
 
 dependencies {
     implementation 'org.slf4j:slf4j-api:1.7.32'
 
     testImplementation 'junit:junit:4.13.1'
+
+    //examplesImplementation 'com.github.mbelling:rpi-ws281x-java:2.0.0-SNAPSHOT' //doesn't work for some reason 🙄
+    examplesImplementation files("${System.properties['user.home']}/.m2/repository/com/github/mbelling/rpi-ws281x-java/2.0.0-SNAPSHOT/rpi-ws281x-java-2.0.0-SNAPSHOT.jar")
```

Comment:
I feel like there is probably a better way to do this, I will think about it and come back to it.

Reply:
<https://github.com/Ro5bert/MCP23S17/blob/bbe9ad027720a2bb4c8de87faffd1073bd7a3bf7/build.gradle#L89C1-L89C72>

I did it differently in the MCP23S17 project: I made a dependency for the jar in the build/libs folder instead of the local maven repo. Seems more straight-forward.